src/Game.cxx

```diff
@@ -43,6 +43,18 @@ Game::Game()
   LOG(LOG_DEBUG) << "Created Game Object";
 }
 
+void Game::quit()
+{
+#ifdef USE_AUDIO
+  m_AudioMixer.stopAll();
+  if (!Settings::instance().audio3DStatus)
```

Comment:
Every if condition should have a brackets even for one command
That's part of the Team code convension